src/components/projects/wbs-details/work-package-container/changes-list/changes-list.tsx

```diff
@@ -20,8 +22,17 @@ const ChangesList: React.FC<ChangesListProps> = ({ changes }) => {
       headerRight={<></>}
       list={changes.map((ic) => (
         <>
-          [<Link to={`${routes.CHANGE_REQUESTS}/${ic.changeRequestId}`}>#{ic.changeRequestId}</Link>
-          ] {ic.detail}
+          [<Link to={`${routes.PROJECTS}/${wbsPipe(ic.wbsNum)}`}>{wbsPipe(ic.wbsNum)}</Link>]{' '}
+          <OverlayTrigger
+            placement="right"
+            overlay={
+              <Tooltip id="tooltip">
+                {fullNamePipe(ic.implementer)} - {datePipe(new Date(ic.dateImplemented))}
```

Comment:
Do you need to make a new Date? I believe ImplementedChange.dateImplemented is already supposed be a Date

Reply:
yeah, I added this while troubleshooting because datePipe wasn't recognizing it as a date

Reply:
why wasn't it? can you screenshot the error?

Reply:
![image](https://user-images.githubusercontent.com/29521172/163734829-65ca95b8-f501-482f-adac-14586a22ae86.png)

Reply:
The transformers were never updated with ImplementedChange.dateImplemented, so it's just a string rn

Reply:
oh I see, I had just assumed the backend was working correctly and didn't check that, I'll go fix it

Reply:
thanks alexander